src/modules/previewpane/powerpreview/powerpreview.h

```diff
@@ -24,20 +24,20 @@ class PowerPreviewModule : public PowertoyModuleIface
         m_previewHandlers(
             { // SVG Preview Hanlder settings object.
               new FileExplorerPreviewSettings(
-                  false,
-                  GET_RESOURCE_STRING(IDS_PREVPANE_SVG_BOOL_TOGGLE_CONTROLL),
+                  true,
+                  L"svg-previewer-toggle-setting",
                   GET_RESOURCE_STRING(IDS_PREVPANE_SVG_SETTINGS_DESCRIPTION),
                   L"{ddee2b8a-6807-48a6-bb20-2338174ff779}",
-                  GET_RESOURCE_STRING(IDS_PREVPANE_SVG_SETTINGS_DISPLAYNAME),
+                  L"SVG Preview Handler",
```

Comment:
Just double checking this doesn't need to be localized

Reply:
This is not an UI display string. It's a registry value data for the preview handler updated the [name of the variable in](https://github.com/microsoft/PowerToys/blob/0d54274c36ac87a24eaaf4eab691ae7dbed70c90/src/modules/previewpane/powerpreview/settings.h#L17) `FileExplorerSettings.h`. There is a separate issue to[ remove the non-ui strings from](https://github.com/microsoft/PowerToys/issues/1646) resource file.